jbpm-human-task/jbpm-human-task-workitems/src/main/java/org/jbpm/services/task/wih/util/HumanTaskHandlerHelper.java

```diff
@@ -145,7 +145,7 @@ public static Deadlines setDeadlines(WorkItem workItem, List<OrganizationalEntit
 			EmailNotification emailNotification = TaskModelProvider.getFactory().newEmialNotification();
 			notifications.add(emailNotification);
 
-			emailNotification.setBusinessAdministrators(businessAdministrators);
+			emailNotification.setBusinessAdministrators(new ArrayList<>(businessAdministrators));
```

Comment:
Just curious if the collection is modified anywhere so we have to do a copy of it...

Reply:
this is required so hibernate will not fail saying there is a shared collection used for several entities.